docs/userGuide/syntax/questions.md

```diff
@@ -217,6 +217,83 @@ correct | `Boolean` | `false` | Whether this option (placed under either a MCQ o
 reason{{slot_info_trigger}}  | `String` | `''` | The explanation markup to display for the option once the answer is checked.
 
 
+#### Fill-in-the-Blanks Questions {.mt-4 .mb-3}
+
+Fill-in-the-blanks questions are specified with the `type="blanks"` attribute.
+
+Unlike MCQ and checkbox questions, answer checking is performed for each blank by providing keywords to check for in the user's answer through the `keywords` attribute in each `q-option`.
+If no keywords are provided, the answer for that blank will always be marked as correct.
+
+<box type="warning" seamless>
+
+Keywords are validated by checking if the keyword matches the user's answer exactly (ignoring case)!
+This works well for some
+<popover header="When does validation work?">cases
+  <span slot="content">
+  When the keywords given are short and specific to the blank (eg. `abstraction`), it increases the chances that the blank will be validated correctly.
+  <br><br>
+  In contrast, something long and vague like `after discussing for a period of time` which can easily be expressed in a different way (eg. `after deliberating for a while`) would likely cause the blank to be validated incorrectly.
+  </span>
+</popover>
+and not others.
+
+</box>
+
+{% set blanksQuestion %}
+<question type="blanks" hint="Google it!">
+
+  ##### German sociologist __________ called the process of simultaneously analyzing the behavior of individuals and the society that shapes that behavior __________.
+
+  <q-option keywords="Norbert Elias, Elias" reason="That's his name!"></q-option>
+  <q-option keywords="figuration"></q-option>
+</question>
+{% endset %}
+
+<include src="codeAndOutput.md" boilerplate>
+<variable name="highlightStyle">html</variable>
+<variable name="code">{{ blanksQuestion }}</variable>
+</include>
+
+You can also specify a `hide-intermediate-result` attribute to hide the result of each blank upon submitting their attempt.
+
+<box type="tip" seamless>
+
+Since the validation is imperfect, the minimum proportion of correct blanks needed for the entire question to be marked as correct can also be changed using the `threshold` attribute.
+
+If you don't want to validate the answer at all, you may set the `threshold` attribute to `0`. Doing so always marks the blank as correct, and users will be able to see the intended answers.
+
+</box>
+
+{% set blanksQuestion2 %}
+<question type="blanks" hint="What properies would you want these database transactions to have?" threshold=0.75 hideIntermediateResult>
```

Comment:
```suggestion
<question type="blanks" hint="What properies would you want these database transactions to have?" threshold=0.75 hide-intermediate-result>
```

Reply:
imo:
- `hide-interim` seems a little too short / vague
- `feedback` seems longer

Some more suggestions:
- `no-intermediate-result` (following @tlylt's `no` suggestion above + combining your original one =P)
- `no-intermediate-hints`
- `no-incorrect-hints`
- `no-partial-hints`

Reply:
oki! will likely go with `no-intermediate-result` so there's no confusion with the `hint` attribute